### PR TITLE
Document more shortcuts

### PR DESCRIPTION
**Pull request by ptman**
I'm probably missing some still

### CODE REVIEW

File: docs/shortcuts.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ The modifier is <kbd>Ctrl</kbd> on Windows & Linux and <kbd>⌘</kbd> on Mac.
 - <kbd>Ctrl</kbd>/<kbd>⌘</kbd>+<kbd>d</kbd> - toggle mic mute
 - <kbd>Ctrl</kbd>/<kbd>⌘</kbd>+<kbd>e</kbd> - toggle video on/off
 - <kbd>Ctrl</kbd>/<kbd>⌘</kbd>+<kbd>k</kbd> - jump to named room
-- <kbd>↑</kbd>/<kbd>↓</kbd> - navigate within room list
+- <kbd>↑</kbd>/<kbd>↓</kbd> - navigate old messages to edit in composer
+- <kbd>↑</kbd>/<kbd>↓</kbd> - next/prev room when focus in room list
+- <kbd>Alt</kbd>+<kbd>↑</kbd>/<kbd>↓</kbd> - scroll timeline up/down ?
```
Comment (anoadragon453) [inline docs/shortcuts.md]: These shortcuts let you resend previous messages you've sent.

```suggestion
- <kbd>Alt</kbd>+<kbd>↑</kbd>/<kbd>↓</kbd> - resend previous messages when the composer is in focus
```

Comment (ptman) [reply docs/shortcuts.md]: Isn't that without Alt? This was taken from something matthew said.

Comment (ptman) [reply docs/shortcuts.md]: https://matrix.to/#/!CSppKHdnjZGxAQKhfE:mozilla-test.modular.im/$g4XgA1J_X7iPrkMtuqFqgaxmC74z3VTWkKEmBIZrPFs?via=mozilla-test.modular.im

Comment (anoadragon453) [reply docs/shortcuts.md]: Up/Down is edit messages and resend. Alt-Up/Down is just copy in the text from previous messages and resend as new ones.

**Comment by ptman** [comment]
ok, didn't work for me, but I trust your knowledge

File: docs/shortcuts.md [hunk 2]
```diff
@@ -6,6 +6,8 @@ The modifier is <kbd>Ctrl</kbd> on Windows & Linux and <kbd>⌘</kbd> on Mac.
 - <kbd>Ctrl</kbd>/<kbd>⌘</kbd>+<kbd>d</kbd> - toggle mic mute
 - <kbd>Ctrl</kbd>/<kbd>⌘</kbd>+<kbd>e</kbd> - toggle video on/off
 - <kbd>Ctrl</kbd>/<kbd>⌘</kbd>+<kbd>k</kbd> - jump to named room
-- <kbd>↑</kbd>/<kbd>↓</kbd> - navigate within room list
+- <kbd>↑</kbd>/<kbd>↓</kbd> - navigate old messages to edit in composer
```
Comment (anoadragon453) [inline docs/shortcuts.md]:
```suggestion
- <kbd>↑</kbd>/<kbd>↓</kbd> - navigate old messages to edit when the composer is in focus
```

**Pull request merged**